db/migrate/20210325171835_improve_coronavirus_announcements_schema.rb

```diff
@@ -0,0 +1,20 @@
+class ImproveCoronavirusAnnouncementsSchema < ActiveRecord::Migration[6.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        # Change from string to allow URL's longer than 255 characters
+        change_column :coronavirus_announcements, :url, :text, null: false
+      end
+
+      dir.down do
+        change_column :coronavirus_announcements, :url, :string
+      end
+    end
+
+    # These fields are all required so shouldn't allow null
+    change_column_null :coronavirus_announcements, :coronavirus_page_id, false
+    change_column_null :coronavirus_announcements, :title, false
+    change_column_null :coronavirus_announcements, :url, false
+    change_column_null :coronavirus_announcements, :published_at, false
```

Comment:
I am confused... shouldn't `published_at` be able to be `null`? It looks fine in the schema below.

Reply:
Oh sorry - that was left over from a quick commit I did so I could undo setting that to null false in integration: https://github.com/alphagov/collections-publisher/commit/f4b03223a777341bc1fc40f7548559a04cfb6d89

I've removed that now.

Reply:
👍